tests/apps/courses/test_templates_course_detail.py

```diff
@@ -138,14 +138,14 @@ def test_course_cms_draft_content(self):
         for organization in [organization1, organization2]:
             self.assertContains(
                 response,
-                '<li class="course-detail__organizations__item">{:s}</li>'.format(
+                '<li class="course-detail__content__organizations__item">{:s}</li>'.format(
                     organization.extended_object.get_title()
                 ),
                 html=True,
             )
         self.assertContains(
             response,
-            '<li class="course-detail__organizations__item--draft">{:s}</li>'.format(
+            '<li class="course-detail__content__organizations__item--draft">{:s}</li>'.format(
```

Comment:
Maybe you can add some checks since you changed the template so much? Maybe the presence of organization_main at the right place for example.